src/components/structures/UserSettings.js

```diff
@@ -60,10 +61,21 @@ module.exports = React.createClass({
             threePids: [],
             phase: "UserSettings.LOADING", // LOADING, DISPLAY
             email_add_pending: false,
+            vectorVersion: null,
         };
     },
 
     componentWillMount: function() {
+        if (PlatformPeg.get()) {
+            PlatformPeg.get().getAppVersion().done((appVersion) => {
```

Comment:
no, it's not.

`done` will handle rejected promises. I'm talking about exceptions thrown synchronously by `getAppVersion`.